sdk/java/logbat-sdk/src/main/java/info/logbat/config/LogBatConfigLoader.java

```diff
@@ -0,0 +1,158 @@
+package info.logbat.config;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import org.yaml.snakeyaml.Yaml;
+
+/**
+ * Utility class for loading LogBat configuration from various file formats. Supports YAML (.yml,
+ * .yaml) and Properties (.properties) file formats. Configuration files are expected to be in the
+ * classpath.
+ *
+ * @author KyungMin Lee <a href="https://github.com/tidavid1">GitHub</a>
+ * @version 0.1.2
+ * @since 0.1.2
+ */
+@SuppressWarnings("unchecked")
+public class LogBatConfigLoader {
+
+    private static final Map<String, String> CONFIG_MAP = new HashMap<>();
```

Comment:
이렇게 하면 기존 코드를 변경하지 않고, 하나의 키에 대해 중복 값을 추가하려는 경우 error를 던지도록 할 수 있어겠어요!

```suggestion
    private static final Map<String, String> CONFIG_MAP = new HashMap<>(){
        @Override
        public String put(String key, String value) {
            if (this.containsKey(key)) {
                if (this.get(key).equals(value)) {
                    return value;
                } else {
                    throw new IllegalArgumentException("Duplicate key: " + key + " with different values " + value + " and " + this.get(key));
                }
            }
            return super.put(key, value);
        }
        
        @Override
        public void putAll(Map<? extends String, ? extends String> m) {
            for (Map.Entry<? extends String, ? extends String> entry : m.entrySet()) {
                this.put(entry.getKey(), entry.getValue());
            }
        }
    };
```